pep-0461.txt

```diff
@@ -55,7 +53,7 @@ All the numeric formatting codes (``d``, ``i``, ``o``, ``u``, ``x``, ``X``,
 ``e``, ``E``, ``f``, ``F``, ``g``, ``G``, and any that are subsequently added
 to Python 3) will be supported, and will work as they do for str, including
 the padding, justification and other related modifiers (currently ``#``, ``0``,
-``-``, `` `` (space), and ``+`` (plus any added to Python 3)).  The only
+``-``, Space, and ``+`` (plus any added to Python 3)).  The only
```

Comment:
Lower case, not a proper noun? 
```suggestion
``-``, space, and ``+`` (plus any added to Python 3)).  The only
```

Reply:
Agree on lower case, but isn't ``` `` `` ``` valid syntax?

A

Reply:
Apparently not, the current rendering just shows the backticks: https://peps.python.org/pep-0461/#interpolation

Reply:
Yeah, I had tried it and it didn't work. I also tried non-breaking spaces, but they didn't work either. I also considered some symbolic representation of a space, but that would likely be confusing and mean the literal isn't actually a literal. There's also `|word|` replacements, but as far as I could tell they don't work inside literal blocks. I would consider just using the prose `" "` but it would be inconsistent formatting with the others (though to be fair, so is this to some extent). I also consulted [this SO issue](https://stackoverflow.com/questions/31304522/how-to-document-a-single-space-character-within-a-string-in-rest-sphinx) but wasn't able to find anything useful.

If you can come up with anything, I can try it, but I'm not sure how much effort is worth it for an old PEP.

Reply:
Made the tweak to lowercase to the appropriate commit while I was rebasing to test on the latest `main` :+1: